README.md

```diff
@@ -3,6 +3,11 @@ Backend API for Moo using Nest framework (NodeJS + TypeScript + PostgreSQL)
 
 ![NodeJS Workflow](https://github.com/scaleracademy/twitter-backend-node/actions/workflows/nodejs.yml/badge.svg)
 
+## Prerequisites
+```
+1. Node
+2. Docker
```

Comment:
Docker is not a prerequisite.

You can remove it or mark it as optional.

Anybody can use their own database that they have, instead of running docker compose.